avanza/constants.py

```diff
@@ -91,7 +96,7 @@ class Route(enum.Enum):
     ACCOUNT_OVERVIEW_PATH = '/_mobile/account/{}/overview'
     ACCOUNTS_POSITIONS_PATH = '/_api/position-data/positions'
     AUTHENTICATION_PATH = '/_api/authentication/sessions/usercredentials'
-    CHARTDATA_PATH = '/_mobile/chart/orderbook/{}'
+    CHARTDATA_PATH = '/_api/price-chart/stock/{}'
```

Comment:
The `get_chart_data` which uses this api url takes an order book id which can be anything from a stock, a fund, a warrant.

Does the new url work with all types of instruments or just stocks?

Reply:
I have just tested it works for stock, warrants, index and funds.